src/op_mode/ssh.py

```diff
@@ -60,3 +61,40 @@ def show_fingerprints(raw: bool, ascii: bool):
             return []
         else:
             return "No SSH server public keys are found."
+
+def show_dynamic_protection(raw: bool):
+    config = ConfigTreeQuery()
+    if not config.exists("service ssh dynamic-protection"):
+        raise vyos.opmode.UnconfiguredSubsystem("SSH server dynamic-protection is not enabled.")
+
+    attackers = []
+    try:
+        # IPv4
+        attackers = attackers + json.loads(cmd("sudo nft -j list set ip sshguard attackers"))["nftables"][1]["set"]["elem"]
```

Comment:
We try to not have `sudo` in the op mode scripts as long as this is not ultimately required. Please rather add the `sudo` call to the OP mode XML template.

Reply:
I see calls to sudo in many other of the Python op_mode scripts... Also, this was accepted as is to 1.5, do we really need to change it? If so, it should be updated there as well.

Reply:
We have a lot of code accepted in 1.4 and 1.5 that we later iterate on. No big deal.

I will append a commit with the changes to this PR here

Reply:
The remaining calls to `sudo` are there b/c those scripts have yet most likely not been converted to the new style op-mode scripts.